protocol-units/bridge/contracts/src/AtomicBridgeInitator.sol

```diff
@@ -0,0 +1,137 @@
+import {console} from "forge-std/Test.sol";
+import "./IAtomicBridgeInitiator.sol";
+import "./WETH/interfaces/IWETH10.sol";
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+
+contract AtomicBridgeInitiator is IAtomicBridgeInitiator {
+    struct BridgeTransfer {
+        uint256 amount;
+        address originator;
+        address recipient;
+        bytes32 hashLock;
+        uint timeLock;
+        bool exists;
```

Comment:
what is `exists` used for?  We set it to true when we create a transfer which is then stored as a keyed value, hence it exists

Reply:
Yes I had removed this field for the same reason. It was in the @cde8's original design. But it does allow for this nice error message on L73. Without it user would get a generic revert error. But I'm happy to remove it if we want. 
`require(bridgeTransfer.exists, "Bridge transfer does not exist");`